mmda/types/box.py

```diff
@@ -28,15 +28,13 @@ class Box:
     h: float
     page: int
 
-    def to_json(self) -> List:
-        return [self.l, self.t, self.w, self.h, self.page]
-        # return dict(l=self.l, t=self.t, w=self.w, h=self.h, page=self.page)
+    def to_json(self) -> Dict[str, float]:
+        return {'left': self.l, 'top': self.t, 'width': self.w, 'height': self.h, 'page': self.page}
 
     @classmethod
-    def from_json(cls, box_coords: List) -> "Box":
-        l, t, w, h, page = box_coords
+    def from_json(cls, box_dict: Dict[str, float]) -> "Box":
+        l, t, w, h, page = [box_dict[key] for key in ['left', 'top', 'width', 'height', 'page']]
         return Box(l=l, t=t, w=w, h=h, page=page)
```

Comment:
```suggestion
        return Box(l=box_dict['left'], t=box_dict['top'], w=box_dict['width'], h=box_dict['height'], page=box_dict['page'])
```
slightly cleaner to look up, I think it's also faster.